challenges/objects-arrays.js

```diff
@@ -97,23 +145,45 @@ The zoos need a list of all their animal's names (names only, not scientific) co
 
 */
 
-const lowerCase = [];
+//const lowerCase = [];
+const lowerCase = zooAnimals.map((name) => {
+  return name.animal_name.toLowerCase();
+});
 console.log(lowerCase); 
 
+
+
 /* Request 3: .filter() 
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const lowerPopulation = [];
-console.log(lowerPopulation);
+const lowerPopulation = zooAnimals.filter(function(animals){
+  return animals.population < 5;
+});
+
+console.log(lowerPopulation.length);
 
 /* Request 4: .reduce() 
 
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
```

Comment:
here is a simpler way to use reduce to solve this 
const populationTotal = zooAnimals.reduce((animal, totalPopulation) => {
  return animal + totalPopulation.population;
}, 0);
console.log(populationTotal);

remember advanced array methods remove the need of using a for loop

Reply:
got it